plugins/inventory/aws_ec2.py

```diff
@@ -87,7 +87,10 @@
 
 # Example using filters, ignoring permission errors, and specifying the hostname precedence
 plugin: aws_ec2
+# The values for profile, access key, secret key and token can be hardcoded like:
 boto_profile: aws_profile
+# or you could use Jinja as:
+# boto_profile: "{{ lookup('env', 'MY_PROJECT_AWS_PROFILE') | default('aws_profile', true) }}"
```

Comment:
```suggestion
# boto_profile: "{{ lookup('env', 'AWS_PROFILE') | default('aws_profile', true) }}"
```

Just for consistency with our 'normal' modules, I think it's worth the example matching what we normally support.

Reply:
@gabops Please add a different example here, since inventory plugin supports environment variables.

Reply:
Hello there, it's been a while since I worked on this and I would like to push it forward. Could you please tell me which example would be good to add from your point of view?

Reply:
@bcoca, I remember a similar PR and you said it was not good idea to use Jinja2 in the inventory configuration?

Reply:
It should indeed by `AWS_PROFILE` like @tremble suggested.